File: help/eng_gb/exercicios_da_ficha_3/Ficha03_006/Ficha03_006.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Pythagorean Theorem\n" );
    document.write("Catet a \t: " );// print text
    a = parseFloat(prompt("Catet a \t: " ,"0.0") );
    
    document.write(a + "\n");// print input 
    document.write("Catet b \t: " );// print text
    b = parseFloat(prompt("Catet b \t: " ,"0.0") );
    
    document.write(b + "\n");// print input 
    var c = Math.pow( Math.pow( a , 2 ) + Math.pow( b , 2 ) , 0.5 ) ;
    document.write("Hypotnusa \t: " + c );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:56

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_006/Ficha03_006.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_006 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Pythagorean Theorem\n" );
        System.out.print("Catet a \t: " );
        double a = Double.valueOf(keyb.nextLine());
        System.out.print("Catet b \t: " );
        double b = Double.valueOf(keyb.nextLine());
        double c = Math.pow( Math.pow( a , 2 ) + Math.pow( b , 2 ) , 0.5 ) ;
        System.out.print("Hypotnusa \t: " + c );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:56
